sale_prequotation/sale_prequotation.py

```diff
@@ -869,6 +874,25 @@ def action_sale(self, cr, uid, ids, format_quote, context=None):
         result['domain'] = "[('id','in', [" + ','.join(map(str, quote_ids)) + "])]"
         return result
 
+    def action_new_pq_version(self, cr, uid, ids, context=None):
+        for prequote in self.browse(cr, uid, ids, context=context):
+            pq_new = self.copy(cr, uid, prequote.id, context=context)
+            self.write(cr, uid, self.search(cr, uid, [('id', '=', pq_new)]), {
```

Comment:
why not just self.write(cr, uid, [new_pq_id], context=context)

Reply:
Fixed it